bench/bench.ml

```diff
@@ -128,7 +125,7 @@ let () =
   Path.Build.set_build_dir (Path.Outside_build_dir.of_string "_build");
   let module Scheduler = Dune_engine.Scheduler in
   let config =
-    Dune_engine.Clflags.display := Dune_engine.Display.Quiet;
+    Dune_engine.Clflags.display := Quiet;
```

Comment:
I'm confused: what does this setting affect? Nothing?

Reply:
The confusing part is basically setting `display` to `Quiet` twice in two different places. I can imagine someone tweaking this code 2 years from now changing just one and forgetting to change the other. Can we eliminate this mutable field altogether?

Reply:
This is not really a problem that this PR introduced, so you should feel free to move my comment into the code in the form of a TODO. But since you're changing this code in a series of PRs, I'd appreciate it if you also try to make it better in the long term by simplifying the code (and getting rid of unnecessary mutable state is a good way forward).

Reply:
> I can imagine someone tweaking this code 2 years from now might change just one and forget to change the other and introduce a subtle bug. Can we eliminate this mutable field altogether?

But it would be nice, but I think it's outside the scope of this PR. There's a whole lot of command line arguments that we can pass explicitly into dune_engine but we aren't at the moment.

> The confusing part is basically setting display to Quiet twice in two different places. I can imagine someone tweaking this code 2 years from now might change just one and forget to change the other and introduce a subtle bug

Note that `bench.ml` is a different binary than dune (it runs the benchmark against pat diff). So it just has a hardcoded verbosity setting and changing it doesn't affect anything else.

Reply:
> I'm confused: what does this setting affect? Nothing?

Indeed, this does nothing. Just removes the unnecessary module qualification.

Reply:
Well, it seems like we agree that the current state is pretty unsatisfactory. I'm just proposing to document our agreement in code. (And maybe create an issue asking for help.)